orttraining/orttraining/python/ort_trainer.py

```diff
@@ -199,6 +199,44 @@ def dtype_torch_to_numpy(torch_dtype):
     else:
         raise Exception("Torch type to numpy type mapping unavailable for: " + str(torch_dtype))
 
+class model_loss_cls(torch.nn.Module):
```

Comment:
I am assuming you just moved this piece of code around and no change was performed. Is that right?

Reply:
yes, move it outside the method so that can call `isinstance` with it elsewhere.